packages/components/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - `underline` prop for `Link Pure`
 - `Checkbox Wrapper`, `Radio Button Wrapper`, `Popover` and `Tag Dismissible` supports `theme` prop
 - `Icon` supports `x-small` and `x-large` values of `size` prop
+- `Accordion` `size="medium"` in `compact` variant
```

Comment:
Not sure what this means.

Reply:
Size medium was only available when `compact="false"`. I wrote it a bit different to clarify